ingest/src/main/java/com/microsoft/azure/kusto/ingest/ManagedStreamingIngestClient.java

```diff
@@ -291,7 +292,7 @@ public IngestionResult ingestFromResultSet(ResultSetSourceInfo resultSetSourceIn
     }
 
     @Override
-    public IngestionResult ingestFromStream(StreamSourceInfo streamSourceInfo, IngestionProperties ingestionProperties)
+    public IngestionResult ingestFromStreamImpl(StreamSourceInfo streamSourceInfo, IngestionProperties ingestionProperties)
```

Comment:
Make sure all the impl methods are not public

Reply:
Done